src/ui/theme/recipes/TabsRecipe.ts

```diff
@@ -26,12 +25,118 @@ export const tabsSlotRecipe = defineSlotRecipe({
       },
     },
   },
+
   variants: {
+    size: {
+      sm: {
+        list: {
+          gap: '1',
+        },
+        trigger: {
+          py: '1',
+          px: '2',
+          fontSize: 'xs',
+          borderRadius: 'redesign.sm',
+          fontWeight: 'medium',
+          _selected: {
+            boxShadow: '{shadows.elevation2}',
+          },
+        },
+      },
+      md: {
+        trigger: {
+          py: '1',
+          px: '3',
+          fontSize: 'sm',
+          borderRadius: 'redesign.md',
+          fontWeight: 'medium',
+        },
+      },
+      lg: {
+        trigger: {
+          py: '1',
+          px: '3',
+          fontSize: 'xl',
+          borderRadius: 'redesign.md',
+          fontWeight: 'regular',
+        },
+      },
+      xlg: {
```

Comment:
typically this is written xl. we use this nomenclature elsewhere so we should use the same here